pkg/debian/copyright

```diff
@@ -28,3 +28,27 @@ License: Commercial
  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
  OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  SUCH DAMAGE.
+
+Files: pkg/debian/*
+Copyright: 2009 - 2025 Tarsnap Backup Inc.
```

Comment:
The tarsnap client code is, in general, copyright 2006 onwards.  But `pkg/debian/` was only added in 2009, so I figured that was a better start date.